plugin/src/main/java/com/denizenscript/denizen/scripts/commands/world/TickCommand.java

```diff
@@ -0,0 +1,120 @@
+    //
+    // To make the tick rate as fast as possible for a specific amount of time, use the 'sprint' argument and input the amount of ticks using 'amount'.
+    // The tick rate will increase as much as possible without overloading the server for the amount of specified ticks.
+    // For example, if you want to sprint 200 ticks, the tick rate will run 200 ticks as fast as possible and then return to the previous tick rate.
+    // To stop stepping or sprinting early, use the 'cancel' argument.
+    //
+    // The tick rate resets to 20.0 on server restart.
+    // For information about tick rate arguments, see <@link url https://minecraft.wiki/w/Commands/tick>
+    //
+    // @Warning Be careful, this command will affect plugins that depend on tick rate and may cause them to break. For example, setting the tick rate to 1 will cause the <@link event tick> event to fire once per second.
+    //
+    // @Usage
+    // Use to set the tick rate to 30 ticks per second.
+    // - tick rate amount:30
+    //
+    // @Usage
+    // Use to stop stepping early if the server is currently stepping.
+    // - tick step cancel
+    //
+    // @Usage
+    // Use to reset tick rate.
+    // - tick reset
+    //
+    // @Usage
+    // Use to freeze the server.
+    // - tick freeze
+    //
+    // @Usage
+    // Use to unfreeze the server.
+    // - tick freeze cancel
+    // -->
+
+    public TickCommand() {
+        setName("tick");
+        setSyntax("tick [rate/step/sprint/freeze/reset] (amount:<amount>/cancel)");
+        setRequiredArguments(1, 2);
+        isProcedural = false;
+        autoCompile();
+    }
+
+    public enum TickActions { RATE, STEP, SPRINT, FREEZE, RESET }
+
+    public static void autoExecute(@ArgName("action") @ArgLinear TickActions action,
+                                   @ArgName("amount") @ArgPrefixed @ArgDefaultNull ElementTag amount,
+                                   @ArgName("cancel") boolean cancel) {
+        ServerTickManager tickManager = Bukkit.getServerTickManager();
+        switch (action) {
+            case RATE -> {
+                if (amount == null || !amount.isFloat()) {
+                    throw new InvalidArgumentsRuntimeException("The rate action must have a decimal number input!");
+                }
+                if (amount.asFloat() < 1 || amount.asFloat() > 10000) {
+                    throw new InvalidArgumentsRuntimeException("Invalid input! Tick rate must be a decimal number between 1.0 and 10000.0 (inclusive)!");
+                }
+                tickManager.setTickRate(amount.asFloat());
+            }
+            case STEP -> {
+                if (cancel) {
+                    tickManager.stopStepping();
+                    return;
+                }
+                if (amount == null || !amount.isInt()) {
+                    throw new InvalidArgumentsRuntimeException("The step action must have an integer input!");

```

Comment:
Errors/meta generally use `number`/`decimal number` (i.e. `ElementTag(Number)` and `ElementTag(Decimal)`)